lib/y2lan_restart_common.pm

```diff
@@ -49,7 +49,7 @@ sub initialize_y2lan
 }
 
 sub open_network_settings {
-    type_string "yast2 lan; echo yast2-lan-status-\$? > /dev/$serialdev\n";
+    type_string "Y2DEBUG=1 ZYPP_MEDIA_CURL_DEBUG=1 yast2 lan; echo yast2-lan-status-\$? > /dev/$serialdev\n";
```

Comment:
What's about creating wrapper similar to `x11_start_program`?

Reply:
That would be another step, but I need to wait for https://progress.opensuse.org/issues/46676. I just want to avoid shifting stuff here and there atm. When it will be merged, I will create a wrapper for console-like tests.

Reply:
That PR already has conflicts and some issues, so I would vote for merging this one first, but yeah it's deadlock =)

Reply:
Alright, so do you think I should invest more time to find a place for the wrapper now, or we can leave it as it is for now and create a follow-up ticket? As long as I have not missed any yast2 invocation, we are missing only the wrapper code.

Reply:
I would just simply add it to the `y2logsstep.pm` and then we can refactor it to move it to the right place. With you proposal we will have to do work twice, once adding env vars and then moving them to the wrapper.

Reply:
alright!